src/__tests__/utils/password.test.ts

```diff
@@ -0,0 +1,18 @@
+import { checkHash, hashFunction } from '../../utils/password';
+
+describe('testing the hashFunction', () => {
+  it('should compare the password hashed and without', () => {
+    expect(hashFunction('password')).not.toBe('password');
+  });
+
+  it('should compare the length of 2 hashed passwords', async () => {
+    const hashed1 = await hashFunction('password1');
```

Comment:
Might be even better to generate 2 strings with randomized length instead of hardcoding them